tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AutoInstrumentationExtensions.cs

```diff
@@ -28,5 +30,28 @@ internal static void DisposeWithException(this Scope scope, Exception exception)
                 }
             }
         }
+
+        public static bool TryGetAssemblyFileVersionFromType(Type type, out Version? version)
```

Comment:
Non-blocking comment, but do we have a better place for helper/static methods like this? The file/class name ending with `Extensions` implies it contains only extension methods. (I would even rename this one to `ScopeExtensions`, heh)

Reply:
I was planning to transform this to an extension as well (without the renaming)

Reply:
done in [4b80233](https://github.com/DataDog/dd-trace-dotnet/pull/5958/commits/4b8023366e77a0e451f4b8c65bc155b22b8e7d4f)